app/controllers/AppealsController.scala

```diff
@@ -54,9 +54,9 @@ class AppealsController @Inject()(appConfig: AppConfig,
                                   cc: ControllerComponents)(implicit ec: ExecutionContext)
   extends BackendController(cc) with FeatureSwitching {
 
-  private def getAppealDataForPenalty(penaltyId: String,
-                                      enrolmentKey: String, penaltyType: AppealTypeEnum.Value)(implicit hc: HeaderCarrier): Future[Result] = {
-    if (!isEnabled(UseAPI1812Model)) {
+  private def getAppealDataForPenalty(penaltyId: String, enrolmentKey: String,
+                                      penaltyType: AppealTypeEnum.Value, useNewApiModel: Boolean)(implicit hc: HeaderCarrier): Future[Result] = {
```

Comment:
I think we can initialise the boolean value 'useNewApiModel' to false.

Reply:
It gets set in my appeals frontend PR https://github.com/hmrc/penalties-appeals-frontend/pull/300/files#diff-40b59660a23903e3ce6c42593b69e162e754b949c3fe120777db7e159a5ea529R42